cr.h

```diff
@@ -1130,8 +1135,7 @@ static cr_failure cr_signal_to_failure(int sig) {
 #endif
 
 static void cr_plat_init() {
-#ifdef __MINGW32__
-    CR_TRACE
```

Comment:
Keep the CR_TRACE please

Reply:
I removed CR_TRACE there because since `cr_plat_init `is called everywhere (to not lose the signal subscription) it would spam the log every frame with CR_TRACE, not sure how to fix it :/

Reply:
Right, that is fine then. :)

Reply:
@fungos  I think we will have to revert this commit as I am know testing with Mingw32 + clang and having compile errors.

It works on native Clang though with no problem. the issue comes with mingw + clang combination

Reply:
what is the issue with mingw32 + clang?

Reply:
> what is the issue with mingw32 + clang?

as soon as  I installed the toolchain I got compile errors, __try is not supported with mingw + clang as well as __builtin_longjmp is not available. So it causes confusion between #ifdef __MINGW32__ && defined(__clang__) 

I am very sorry but I think it would be better to revert for now. Current Clang support works only when it is installed on Windows via MSVC linking. i.e clang msvc but is broken when used with mingw32 on Windows

Reply:
@fungos  please consider this PR https://github.com/fungos/cr/pull/69/commits/f750a8fc4493a4f04a62ba583b8bdf1c8c27752c

Reply:
no worries, I need fix CI and put more compilers to be tested